batch/batch/front_end/templates/batch.html

```diff
@@ -46,7 +46,7 @@
 
       {% call collapsible_li(true, 'Jobs', batch['n_jobs']) %}
       {{ kv_table({
-      'Pending': batch['n_jobs'] - batch['n_completed'],
+      'Incomplete (Blocked, Queued or Running)': batch['n_jobs'] - batch['n_completed'],
```

Comment:
I'm like this. To me, 'Pending' implies 'not running', so this improves clarity.